src/ring_theory/localization.lean

```diff
@@ -1163,44 +1163,61 @@ integral_domain_localization (le_non_zero_divisors_of_domain (by simpa only [] u
 end localization_map
 
 section at_prime
+
+namespace localization_map
+
+variables (I : ideal R) [hI : I.is_prime] (f : at_prime S I)
+include hI
+
+lemma at_prime.is_unit_to_map_iff (x : R) :
+  is_unit (f.to_map x) ↔ x ∈ I.prime_compl :=
+⟨λ h hx, (f.is_prime_of_is_prime_disjoint I hI disjoint_compl_left).ne_top $
+  ideal.eq_top_of_is_unit_mem (ideal.map f.to_map I) (ideal.mem_map_of_mem hx) h,
```

Comment:
Dot notation lets you write this:
```suggestion
  (I.map f.to_map).eq_top_of_is_unit_mem (ideal.mem_map_of_mem hx) h,
```

Reply:
Thanks!
I will add your suggestion to #7519.